PUI/PySide6/window.py

```diff
@@ -43,6 +44,8 @@ def update(self, prev=None):
     def addChild(self, idx, child):
         if isinstance(child, MenuBar):
             self.ui.setMenuBar(child.outer)
+        elif isinstance(child, ToolBar):
+            self.ui.addToolBar(child.outer)
```

Comment:
ToolBar can be attached to multiple area, for now it will be attach at top of window by default.